### PR TITLE
Display CPU Temperature

### PR DESCRIPTION
**Pull request by gfelisberto**
**Issue**

- Link:
- Summary:

Fetch the CPU temperature to the diagnostics object and also display in the diagnostics page.

### CODE REVIEW

File: hw_diag/tasks.py [hunk 1]
```diff
@@ -30,6 +31,7 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['last_updated'] = now.strftime("%H:%M UTC %d %b %Y")
 
     get_ethernet_addresses(diagnostics)
+    get_temperatures(diagnostics)
```
Comment (marvinmarnold) [inline hw_diag/tasks.py:34]: Can you rebase on https://github.com/NebraLtd/hm-diag/pull/337 and convert this into a Diagnostic?

File: hw_diag/templates/diagnostics_page.html [hunk 2]
```diff
@@ -148,6 +148,14 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                     <td class="text-danger text-right">N/A</td>
                   {% endif %}
                 </tr>
+                <tr>
+                  <td>CPU Temperature</td>
+                  {% if diagnostics.CPU0 %}
+                  <td class="text-right">{{ diagnostics.E0 }} °C</td>
```
Comment (marvinmarnold) [inline hw_diag/templates/diagnostics_page.html:154]: I think you meant `CPU0`. It should also be further indented.

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
Thanks for your contribution. Some changes requested.

File: hw_diag/tasks.py (hunk 1 above)
Comment (marvinmarnold) [reply hw_diag/tasks.py:34]: Sorry, rebasing on #337 doesn't make sense. But please look at the DiagnosticReport pattern that is being introduced there and copy here.

Comment (gfelisberto) [reply hw_diag/tasks.py:34]: I have almost zero Python skills and could not follow the process

**Pull request closed without merging**